README.adoc

```diff
@@ -476,6 +476,15 @@ This means, for example, that using polling on file mounted secret sources does
 **Notes**:
 - Properties under *spring.cloud.kubernetes.reload.* should not be used in config maps or secrets: changing such properties at runtime may lead to unexpected results;
 - Deleting a property or the whole config map does not restore the original state of the beans when using the *refresh* level.
+- You need to have the reload actuator dependency in your pom file for reload and the reload actuator enabled `management.endpoint.restart.enabled=true` in your `bootstrap.yaml` or `application.yaml`:
```

Comment:
I dont think specifically we need to set `management.endpoint.restart.enabled=true`.  Maybe if you change to `spring.cloud.kubernetes.reload.strategy=restart_context` you might?  Is that what you did.  We should just need to enable the refresh endpoint by default.
```
management:
  endpoints:
    web:
      exposure:
        include: refresh
```

Reply:
I've tried that.

```
spring.cloud.kubernetes.reload.strategy: refresh
#management.endpoint.restart.enabled: true
management.endpoints.web.exposure.include: refresh
```

gives me the following error during startup

```
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.springframework.cloud.context.restart.RestartEndpoint' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
```

Reply:
@paulczar did you include `spring-boot-starter-actuator`?  Also can you try
```
management:
  endpoint:
    restart:
      enabled: true
```

Reply:
yeah both those are what the PR is adding to the docs :)